monai/inferers/utils.py

```diff
@@ -255,7 +255,11 @@ def sliding_window_inference(
                             "Tips: if overlap*roi_size*zoom_scale is an integer, it usually works."
                         )
                     original_idx_zoom[axis] = slice(int(zoomed_start), int(zoomed_end), None)
-                importance_map_zoom = resizer(importance_map.unsqueeze(0))[0].to(compute_dtype)
+                importance_map_zoom = (
+                    resizer(importance_map.unsqueeze(0))[0].to(compute_dtype)
```

Comment:
thanks for looking into this issue, for the resize functional, I think we should also fix the logic when orig_size==out_size? (this is untested):
```py
    if transform_info.get(TraceKeys.LAZY_EVALUATION, False):
        if anti_aliasing and transform_info.get(TraceKeys.LAZY_EVALUATION, False):
            warnings.warn("anti-aliasing is not compatible with lazy evaluation.")
        return out.copy_meta_from(meta_info) if isinstance(out, MetaTensor) else meta_info
    if tuple(convert_to_numpy(orig_size)) == out_size:
        return out.copy_meta_from(meta_info) if isinstance(out, MetaTensor) else out
```

Reply:
Have changed the logic in `Resize` and added a unit test for `set_track_meta(False)`.